doc/plugins.md

```diff
@@ -166,8 +170,11 @@ See [Creating plugins][create] below.
     — make sure at most one top-level heading exists
 *   [`remark-numbered-footnote-labels`](https://github.com/jackfletch/remark-numbered-footnote-labels)
     — label footnotes with numbers
-*   [`remark-oembed`](https://github.com/agentofuser/remark-oembed)
+*   [`@agentofuser/remark-oembed`](https://github.com/agentofuser/remark-oembed)
     — transform URLs for youtube, twitter, etc. embeds
+*   [`remark-oembed`](https://github.com/sergioramos/remark-oembed)
+    — transform URLs surrounded by newlines into embeds that are loaded 
+    *asynchronously*
```

Comment:
```suggestion
    — transform URLs surrounded by newlines into *asynchronously* loaded embeds
```

A bit shorter, does that make sense to you?

Reply:
It does make sense, thank you for the suggestion 👍